packages/assets-controllers/src/TokenDetectionController.test.ts

```diff
@@ -227,25 +241,15 @@ describe('TokenDetectionController', () => {
     controllerMessenger.clearEventSubscriptions(
       'NetworkController:stateChange',
     );
-  });
-
-  it('should set default config', () => {
-    expect(tokenDetection.config).toStrictEqual({
-      interval: DEFAULT_INTERVAL,
-      selectedAddress: '',
-      disabled: true,
-      chainId: ChainId.mainnet,
-      isDetectionEnabledForNetwork: true,
-      isDetectionEnabledFromPreferences: true,
-    });
+    controllerMessenger.unregisterActionHandler(
```

Comment:
Nit: I'm not sure this or the previous statement is necessary, since we recreate `controllerMessenger` per test, but I suppose it doesn't hurt.

Reply:
Removed both: https://github.com/MetaMask/core/pull/3609/commits/74f74ba8486f62b1132a3c632c4b65b697634c46